ex_unit.exs

```diff
@@ -0,0 +1,27 @@
+# Regular use:
+# `elixir ex_unit.exs`
+#
+# Focused use with autostart on file change:
+# `find ex_unit.exs | FOCUS=1 entr -c elixir ex_unit.exs`
+
+focus_options = if System.get_env("FOCUS") == "1", do: [exclude: :test, include: :focus], else: []
```

Comment:
Adding this option to provide a bit more value compared to the `ExUnit` intro documentation.